samples/snippets/src/main/java/beta/video/StreamingAutoMlObjectTracking.java

```diff
@@ -0,0 +1,121 @@
+
+    try (StreamingVideoIntelligenceServiceClient client =
+        StreamingVideoIntelligenceServiceClient.create()) {
+
+      Path path = Paths.get(filePath);
+      byte[] data = Files.readAllBytes(path);
+      // Set the chunk size to 5MB (recommended less than 10MB).
+      int chunkSize = 5 * 1024 * 1024;
+      int numChunks = (int) Math.ceil((double) data.length / chunkSize);
+
+      String modelPath =
+          String.format("projects/%s/locations/us-central1/models/%s", projectId, modelId);
+
+      System.out.println(modelPath);
+
+      StreamingAutomlObjectTrackingConfig streamingAutomlObjectTrackingConfig =
+          StreamingAutomlObjectTrackingConfig.newBuilder().setModelName(modelPath).build();
+
+      StreamingVideoConfig streamingVideoConfig =
+          StreamingVideoConfig.newBuilder()
+              .setFeature(StreamingFeature.STREAMING_AUTOML_OBJECT_TRACKING)
+              .setAutomlClassificationConfig(streamingAutomlObjectTrackingConfig)
+              .build();
+
+      BidiStream<StreamingAnnotateVideoRequest, StreamingAnnotateVideoResponse> call =
+          client.streamingAnnotateVideoCallable().call();
+
+      // The first request must **only** contain the audio configuration:
+      call.send(
+          StreamingAnnotateVideoRequest.newBuilder().setVideoConfig(streamingVideoConfig).build());
+
+      // Subsequent requests must **only** contain the audio data.
+      // Send the requests in chunks
+      for (int i = 0; i < numChunks; i++) {
+        call.send(
+            StreamingAnnotateVideoRequest.newBuilder()
+                .setInputContent(
+                    ByteString.copyFrom(
+                        Arrays.copyOfRange(data, i * chunkSize, i * chunkSize + chunkSize)))
+                .build());
+      }
+
+      // Tell the service you are done sending data
+      call.closeSend();
+
+      for (StreamingAnnotateVideoResponse response : call) {
+        StreamingVideoAnnotationResults annotationResults = response.getAnnotationResults();
+
+        for (ObjectTrackingAnnotation objectAnnotations :
+            annotationResults.getObjectAnnotationsList()) {
+
+          String entity = objectAnnotations.getEntity().getDescription();
+          float confidence = objectAnnotations.getConfidence();
+          long trackId = objectAnnotations.getTrackId();
+          System.out.format("%s: %f (ID: %d)\n", entity, confidence, trackId);
+
+          // In streaming, there is always one frame.
+          ObjectTrackingFrame frame = objectAnnotations.getFrames(0);
+          double offset =
+              frame.getTimeOffset().getSeconds() + frame.getTimeOffset().getNanos() / 1e9;
+          System.out.format("Offset: %f\n", offset);
+
+          System.out.println("Bounding Box:");
+          System.out.format("\tLeft: %f\n", frame.getNormalizedBoundingBox().getLeft());
+          System.out.format("\tTop: %f\n", frame.getNormalizedBoundingBox().getTop());
```

Comment:
NIT: To make this a little bit neater, you might capture a reference to the NormalizedBoundingBox object before this block of `println` statements.